<launch>
    <node pkg="prometheus_detection" type="siam_rpn.py" name="siamrpn_tracker" output="screen">
        <param name="camera_topic" value="/prometheus/camera/rgb/image_raw"/>
        <param name="camera_info" value="$(find prometheus_gazebo)/config/camera_config/camera_param_gazebo_monocular.yaml"/>
    </node>
</launch>
